README.md

```diff
@@ -395,6 +395,20 @@ build folder to your path:
 set PATH=C:\STL\out\build\x64\out\bin\amd64;%PATH%
 ```
 
+## Benchmarks
+
+Building of benchmarks can be enabled by setting `BUILD_BENCHMARKS` to true. The tests use the benchmark library so it
+is also required to set`BUILD_TESTING` to true.
+
+Benchmarks live in the `benchmark` subdirectory and can be build and run via the `msvc_standard_libraries_benchmarks`
+target. Currently all tests are merged into a single binary. Individual tests can be run by passing
+`--benchmark_filter=<regex>` (Please check the official
+[documentation](https://github.com/google/benchmark/blob/main/docs/user_guide.md#running-a-subset-of-benchmarks))
+
+Right now the tests depend on an externally installed benchmark library. This can be done via `vcpkg install benchmark`.

```

Comment:
```suggestion
The tests depend on an externally installed benchmark library. This can be done via `vcpkg install benchmark`.
```
Ditto "tests or benchmarks". What can be done?